src/Build.UnitTests/Evaluation/Evaluator_Tests.cs

```diff
@@ -914,6 +914,44 @@ public void MultipleImportsVerifyImportsIncludingDuplicates()
             }
         }
 
+        [DotNetOnlyFact("Tests .NET SDK-only error")]
+        public void ImportWithVSPathThrowsCorrectError()
+        {
+            InvalidProjectFileException ex = Assert.Throws<InvalidProjectFileException>(() =>
```

Comment:
Nit: scope the lambda for the exception down to _just_ the line you expect to throw, so that you don't get a false-positive hit if some other line throws (here that's unlikely to be a problem since you only call into the API proper once, but it's a good rule to follow).